features/facebookImport/src/model/importers/personal-data-importer.js

```diff
@@ -0,0 +1,28 @@
+import { readJSONDataObject } from "./utils/importer-util.js";
+
+export const PROFILE_INFORMATION_FILE_PATH =
+    "profile_information/profile_information.json";
+export const PROFILE_INFORMATION_DATA_KEY = "profile_v2";
+
+export default class PersonalDataImporter {
+    async _readLanguageData(zipFile) {
+        return readJSONDataObject(
+            PROFILE_INFORMATION_FILE_PATH,
+            PROFILE_INFORMATION_DATA_KEY,
+            zipFile
+        );
+    }
+
+    async import({ zipFile, facebookAccount }) {
+        const profileData = await this._readLanguageData(zipFile);
+        const name = {};
+        name.givenName = [
+            profileData.name.first_name,
```

Comment:
So we have the middle name information... It's just a call not to store it, right?

Reply:
You are right. I assumed givenName would be the `" " join` of first and middle name in schema, which is incorrect. It's just firstName. I will fix it!